akka-docs/rst/java/stream/stages-overview.rst

```diff
@@ -1346,6 +1346,16 @@ Merge multiple sources. Prefer one source if all sources has elements ready.
 
 **completes** when all upstreams complete (This behavior is changeable to completing when any upstream completes by setting ``eagerComplete=true``.)
 
+mergePrioritized
+^^^^^^^^^^^^^^^^
+Merge multiple sources. Prefer sources depending on priorities if all sources has elements ready.
```

Comment:
"If a subset of all sources has elements ready the relative priorities for those sources are used to prioritise."